loadbalancer/fadein_test.go

```diff
@@ -72,196 +46,11 @@ func initializeEndpoints(endpointAges []time.Duration, fadeInDuration time.Durat
-					allBuckets[i-1][epi],
-					allBuckets[i][epi],
-				) {
-					t.Error("non-monotonic change", epi, i)
-				}
-			}
-		}
-
-		for i, bucketStats := range allBuckets {
-			var showStats []string
-			for _, epi := range eps {
-				showStats = append(showStats, fmt.Sprintf("%d", bucketStats[epi]))
-			}
-
-			// Print CSV-like output for, where row number represents time and
-			// column represents endpoint. You can visualize it using
-			// ./skptesting/run_fadein_test.sh from the skipper repo root.
-			t.Log("CSV " + fmt.Sprintf("%d,", i) + strings.Join(showStats, ","))
-		}
-	})
-}
-
-func newConsistentHashForTest(endpoints []string) routing.LBAlgorithm {
-	// The default parameter 100 is too small to get even distribution
-	return newConsistentHashInternal(endpoints, 1000)
-}
-
-func TestFadeIn(t *testing.T) {
-	old := 2 * fadeInDuration
-	testFadeIn(t, "round-robin, 0", newRoundRobin, old, old)
-	testFadeIn(t, "round-robin, 1", newRoundRobin, 0, old)
-	testFadeIn(t, "round-robin, 2", newRoundRobin, 0, 0)
-	testFadeIn(t, "round-robin, 3", newRoundRobin, old, 0)
-	testFadeIn(t, "round-robin, 4", newRoundRobin, old, old, old, 0)
-	testFadeIn(t, "round-robin, 5", newRoundRobin, old, old, old, 0, 0, 0)
-	testFadeIn(t, "round-robin, 6", newRoundRobin, old, 0, 0, 0)
-	testFadeIn(t, "round-robin, 7", newRoundRobin, old, 0, 0, 0, 0, 0, 0)
-	testFadeIn(t, "round-robin, 8", newRoundRobin, 0, 0, 0, 0, 0, 0)
-	testFadeIn(t, "round-robin, 9", newRoundRobin, fadeInDuration/2, fadeInDuration/3, fadeInDuration/4)
-
-	testFadeIn(t, "random, 0", newRandom, old, old)
-	testFadeIn(t, "random, 1", newRandom, 0, old)
-	testFadeIn(t, "random, 2", newRandom, 0, 0)
-	testFadeIn(t, "random, 3", newRandom, old, 0)
-	testFadeIn(t, "random, 4", newRandom, old, old, old, 0)
-	testFadeIn(t, "random, 5", newRandom, old, old, old, 0, 0, 0)
-	testFadeIn(t, "random, 6", newRandom, old, 0, 0, 0)
-	testFadeIn(t, "random, 7", newRandom, old, 0, 0, 0, 0, 0, 0)
-	testFadeIn(t, "random, 8", newRandom, 0, 0, 0, 0, 0, 0)
-	testFadeIn(t, "random, 9", newRandom, fadeInDuration/2, fadeInDuration/3, fadeInDuration/4)
-
-	testFadeIn(t, "consistent-hash, 0", newConsistentHashForTest, old, old)
-	testFadeIn(t, "consistent-hash, 1", newConsistentHashForTest, 0, old)
-	testFadeIn(t, "consistent-hash, 2", newConsistentHashForTest, 0, 0)
-	testFadeIn(t, "consistent-hash, 3", newConsistentHashForTest, old, 0)
-	testFadeIn(t, "consistent-hash, 4", newConsistentHashForTest, old, old, old, 0)
-	testFadeIn(t, "consistent-hash, 5", newConsistentHashForTest, old, old, old, 0, 0, 0)
-	testFadeIn(t, "consistent-hash, 6", newConsistentHashForTest, old, 0, 0, 0)
-	testFadeIn(t, "consistent-hash, 7", newConsistentHashForTest, old, 0, 0, 0, 0, 0, 0)
-	testFadeIn(t, "consistent-hash, 8", newConsistentHashForTest, 0, 0, 0, 0, 0, 0)
-	testFadeIn(t, "consistent-hash, 9", newConsistentHashForTest, fadeInDuration/2, fadeInDuration/3, fadeInDuration/4)
-}
-
-func testFadeInLoadBetweenOldEps(
```

Comment:
This test is removed, but no one is created instead. Please consider moving this test to the proxy package as well.